src/nupic/bindings/experimental.i

```diff
@@ -173,129 +186,152 @@ using namespace nupic;
 
     def activateCells(self,
                       activeColumns,
-                      prevActiveExternalCellsBasal,
-                      prevActiveExternalCellsApical,
+                      reinforceCandidatesExternalBasal=(),
```

Comment:
These empty tuple default values are not consistent with the docstrings below.  I would have expected the type to be "sequence" (e.g. https://docs.python.org/2.4/lib/typesseq.html), or the default value be `None`.